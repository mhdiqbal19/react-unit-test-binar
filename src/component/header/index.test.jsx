import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import Header from './index'

Enzyme.configure({ adapter: new Adapter() })

test("Test Case For header", () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot();
})

