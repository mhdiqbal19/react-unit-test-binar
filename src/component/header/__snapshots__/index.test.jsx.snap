// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Case For header 1`] = `ShallowWrapper {}`;

exports[`Test Case For header should render header 1`] = `ShallowWrapper {}`;
